frontend/src/pages/project/Settings/dialogs/ExportProjectDialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <ff-dialog ref="dialog" header="Export Project" confirm-label="Export Project" @confirm="confirm()">
+    <ff-dialog ref="dialog" header="Copy Project" confirm-label="Copy Project" @confirm="confirm()">
```

Comment:
```suggestion
    <ff-dialog ref="dialog" header="Export Project" confirm-label="Export Project" @confirm="confirm()">
```

Again reverting as Export is the correct way to describe this (if it gets re-enabled). We should schedule a review of this feature for after Thursday.